docs/change_log/2017-07-19-breaking-change-version-6.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Breaking Change: Version 6"
+date: "2017-07-19T07:00:00.000Z"
+breaking: false
```

Comment:
Shouldn't changelog entries that have "Breaking Change" in the title be marked as breaking ones / have breaking set to `true`?

Reply:
Yeah, the parser that pulled this out wasn't perfect.  Did a few spot checks and updated them - do let me know if you find others!